daemon/containerd/image_tag.go

```diff
@@ -25,8 +25,8 @@ func (i *ImageService) TagImage(ctx context.Context, imageID image.ID, newTag re
 	}
 
 	is := i.client.ImageService()
-	_, createErr := is.Create(ctx, newImg)
-	if createErr != nil {
```

Comment:
For those wondering why this special name was here; it was initially used in https://github.com/moby/moby/blob/47569baa3bc35f2764717afbf7ecd7f6848c15c9/daemon/containerd/image_tag.go#L65-L71

```go
	// If the first creation attempt failed, and we deleted the previous image, try again.
	if createErr != nil {
		if _, err = is.Create(ctx, newImg); err != nil {
			logger.WithError(err).Error("failed to create an image after deleting old image")
			return errdefs.System(err)
		}
	}
```

Reply:
Yeah, but then it got removed because we decided to use early-return instead.